### DISCUSSION TITLE
middleware/kafka/user-guide/create/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1895] [Announcements]
# middleware/kafka/user-guide/create/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/middleware/kafka/user-guide/create/

**Comment by liyuerich** [comment]
如果把 访问设置 关了，会有什么影响？
